inginious/frontend/templates/layout.html

```diff
@@ -21,7 +21,7 @@
         <!-- INGInious Style -->
         <link href="{{get_homepath(True)}}/static/css/INGInious-rtl.css?v2" rel="stylesheet">
     {% else %}
-        <link href="{{get_homepath(True)}}/static/css/bootstrap.min.css" rel="stylesheet">
+        <link href="{{get_homepath(True)}}/static/css/themes/default.min.css" rel="stylesheet">
         <!-- INGInious Style -->
         <link href="{{get_homepath(True)}}/static/css/INGInious.css?v2" rel="stylesheet">
```

Comment:
Maybe we should also provide a specific version of this file for the different themes ?